projects/packages/forms/tests/php/contact-form/Contact_Form_Block_Test.php

```diff
@@ -57,15 +61,175 @@ public static function data_provider_test_register_child_blocks() {
 		return array(
 			'jetpack/input'   => array(
 				'jetpack/input',
+				array(
+					'__experimentalBorder' => array(
+						'color'  => true,
+						'radius' => true,
+						'style'  => true,
+						'width'  => true,
+					),
+					'color'                => array(
+						'text'       => true,
+						'background' => true,
+						'gradient'   => true,
+					),
+					'typography'           => array(
+						'fontSize'                     => true,
+						'lineHeight'                   => true,
+						'__experimentalFontFamily'     => true,
+						'__experimentalFontWeight'     => true,
+						'__experimentalFontStyle'      => true,
+						'__experimentalTextTransform'  => true,
+						'__experimentalTextDecoration' => true,
+						'__experimentalLetterSpacing'  => true,
+					),
+				),
 			),
 			'jetpack/label'   => array(
 				'jetpack/label',
+				array(
+					'color'      => array(
+						'text'       => true,
+						'background' => true,
```

Comment:
```suggestion
						'background' => false,
```

It looks like the server side registration is out of whack with the editor registration and the background support for labels should be `false`.